### PR TITLE
added note about PowerShell 7 compatibility

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/2151

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @e0i, for your information

Thank you

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

Peer review needed for this PR. Thanks.

**Review by get-itips — APPROVED** [review]

**Comment by tseward** [comment]
Has this note been added to all modules listed in the compat article?

File: docset/windows/addsadministration/ActiveDirectory.md [hunk 1]
```diff
@@ -26,6 +26,8 @@ The Active Directory module for Windows PowerShell is a PowerShell module that c
 
 If you don't have the Active Directory module installed on your machine, you need to download the correct Remote Server Administration Tools (RSAT) package for your OS.  If you're running Windows 7, you will also need to run the `import-module ActiveDirectory` command from an elevated PowerShell prompt. For more detail, see [RSAT for Windows operating systems](https://support.microsoft.com/help/2693643/remote-server-administration-tools-rsat-for-windows-operating-systems). Starting with Windows 10 October 2018 Update, RSAT is included as a set of Features on Demand right from Windows 10. Now, instead of downloading an RSAT package you can just go to Manage optional features in Settings and click Add a feature to see the list of available RSAT tools. Select and install the specific RSAT tools you need. To see installation progress, click the Back button to view status on the Manage optional features page.
 
+If you want to use this module in PowerShell 7, please see [PowerShell 7 module compatibility](https://docs.microsoft.com/powershell/scripting/whats-new/module-compatibility?view=powershell-7).
```
Comment (dariomws) [inline docset/windows/addsadministration/ActiveDirectory.md:29]:
```suggestion
If you want to use this module in PowerShell 7, please see [PowerShell 7 module compatibility](https://docs.microsoft.com/powershell/scripting/whats-new/module-compatibility).
```

I think it is always a good idea to remove it if it is not necessary.

**Review by dariomws — APPROVED** [review]

**Comment by MaratMussabekov** [comment]
Hello @tseward ,
**Has this note been added to all modules listed in the compat article?** - no it hasn't...
I created this PR to cover https://github.com/MicrosoftDocs/windows-powershell-docs/issues/2151
Is there any way to add this note in bulk?

Thank you

**Comment by tseward** [comment]
You can create a single PR that covers all files that need this update. That should reduce the workload since it would be one large review/commit.

**Pull request merged**